docs/configuring_prometheus.md

```diff
@@ -0,0 +1,36 @@
+# Configuring Prometheus for better Performance with Promscale
+
+This document mentions details about configuring Prometheus's remote-write settings to maximize performance
+from Promscale.
+
+Prometheus is a monitoring tool, that scrapes targets and sends scraped metric data to remote-storage systems
+like Promscale. The samples are pushed by Prometheus through the remote-write component of Prometheus. You
+can configure the remote-write component based on the requirements and tune the component accordingly. Details
+related to the remote-write configuration can be found [here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
+
+Remote-write performance is mainly dependent on `queue_config`. Changing values in `queue_config` will allow
+the component to adjust to different scenarios but ideally they should be:
+
+### capacity
+
+This corresponds to the maximum number of samples that the queues in remote-write shards can hold. If you have
+a high-throughput environment, capacity of queues should be `10000` at least.
+
+### minimum number of shards
+
+Shards are they actual parts of remote-write component that push data to remote-write endpoint (concurrently). More the number of
+shards, more would be the parallel requests to remote-write. Since Promscale is optimized for concurrent inserts,
+it will perform better with more shards. Ideally, you can set `min_shards: 4`. However, you should also keep in mind that increasing shards by
+large values can impact the memory usage significantly.
```

Comment:
Doesn't the part about memory apply to max_shards more than min shards? Thus I remove from the changes 
```suggestion
Shards are they actual parts of remote-write component that push data to remote-write endpoint.
Multiple shards send data concurrently, Thus, the number of shards sets the amount of parallel requests
sent to the remote-write endpoint. Since Promscale is optimized for concurrent inserts,
it will perform better with more shards. Ideally, you can set `min_shards: 4`.
```